commit/merkleroot/outcome.go

```diff
@@ -112,10 +117,17 @@ func reportRangesOutcome(
 		return offRampNextSeqNums[i].ChainSel < offRampNextSeqNums[j].ChainSel
 	})
 
+	if observedRMNRemoteConfig[dstChain].IsEmpty() {
+		lggr.Warn("RMNRemoteConfig is nil")
```

Comment:
Isn't it supposed to be empty in some specific states?
If yes, let's only `Warn` in the state that we actually expect to have it.

Reply:
Valid comment. This method is executed in the context of the `SelectingRangesForReport` state, which mean that we should always have an rmn remote config.

Reply:
Which means that we should always warn if we don't have it.